devel/tilt/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/tilt-dev/tilt 0.18.1 v
+revision            0
+
+categories          devel
+maintainers         nomaintainer
+platforms           darwin
+supported_archs     x86_64
+license             Apache-2
+
+description         Run Kubernetes environments locally, for local development
+
+long_description    Tilt powers multi-service development and makes sure they \
+                    behave! Run tilt up to work in a complete dev environment \
+                    configured for your team. \
+                    \
+                    Tilt automates all the steps from a code change to a new \
+                    process: watching files, building container images, and \
+                    bringing your environment up-to-date. Think docker build \
+                    && kubectl apply or docker-compose up.
+
+homepage            https://tilt.dev/
+
+checksums           rmd160  d2bd6001e581f54986011e10f44a58b68b6f2f80 \
+                    sha256  1a4cb78c4e880157d22d6b8e209d7fc70e77dfef0136929047e5eeee96b6841c \
+                    size    20524516
+
+depends_build       port:yarn port:nodejs12
```

Comment:
The golang portgroup adds go as a build dependency. You should instead use `depends_build-append` to not overwrite this.

https://github.com/macports/macports-ports/blob/1a1667c302671c5f1043338ac0bbd8e44acd3df2/_resources/port1.0/group/golang-1.0.tcl#L143

I think this is what's leading to the error ```sh: /opt/local/bin/go: No such file or directory```